flash_tutorials/electricity_forecasting/electricity_forecasting.py

```diff
@@ -288,7 +289,8 @@ def plot_interpretation(model_path: str, predict_df: pd.DataFrame, parameters: D
 # Now let's look at what it learned:
 
 # %%
-plot_interpretation(trainer.checkpoint_callback.best_model_path, df_energy_daily, datamodule.parameters)
+# Todo: Make sure to uncomment the line below if you want to run predictions and visualize the graph
+# plot_interpretation(trainer.checkpoint_callback.best_model_path, df_energy_daily, datamodule.parameters)
```

Comment:
shouldn't we show them in the rendered notebooks?

Reply:
Prediction of this data is time taking and hence breaks the CI. We decided to just comment it out, and let users run this for their testing.

Unfortunately, adding a trick to make the prediction quicker will break the meaning of this tutorial, so I thought it's better to comment this out - without changing how the prediction code should look like. :)

Reply:
cool